mod_regression/controllers.py

```diff
@@ -63,8 +63,37 @@ def test_view(regression_id):
 
 @mod_regression.route('/test/<regression_id>/delete')
 def test_delete(regression_id):
```

Comment:
I miss authentication here. We don't want to allow anyone to call this and delete the regression test.